lib/inc/cpp-pcp-client/util/logging.hpp

```diff
@@ -28,13 +29,13 @@ LIBCPP_PCP_CLIENT_EXPORT
 void setupLogging(std::ostream &stream,
                   bool force_colorization,
                   std::string const& loglevel_label,
-                  std::shared_ptr<std::ofstream> access_stream = nullptr);
+                  std::shared_ptr<boost::nowide::ofstream> access_stream = nullptr);
```

Comment:
I'd prefer making this a `std::ostream`, it should work and provides a more generic interface.